Build failure real - looks like a test that needs updating:
```
  1) WebForm WebFormSession Should not block requests:
     AssertionError: expected false to equal undefined
      at http://localhost:8000/static/chai/chai.js:210
      at assertEqual (http://localhost:8000/static/chai/chai.js:752)
      at http://localhost:8000/static/chai/chai.js:3854
      at http://localhost:8000/static/chai/chai.js:2507
      at http://localhost:8000/static/cloudcare/js/form_entry/spec/webformsession_spec.js:160
```

--- a/corehq/apps/cloudcare/static/cloudcare/js/form_entry/spec/webformsession_spec.js
+++ b/corehq/apps/cloudcare/static/cloudcare/js/form_entry/spec/webformsession_spec.js
@@ -157,7 +157,7 @@ describe('WebForm', function() {
             // First blocking request
             $.publish('formplayer.' + Formplayer.Const.ANSWER, { answer: sinon.spy() });
 
-            assert.isUndefined(sess.blockingRequestInProgress);
+            assert.isFalse(sess.blockingRequestInProgress);
 
             // Attempt another request
             $.publish('formplayer.' + Formplayer.Const.ANSWER, { answer: sinon.spy() });

--- a/corehq/apps/cloudcare/static/cloudcare/js/form_entry/webformsession.js
+++ b/corehq/apps/cloudcare/static/cloudcare/js/form_entry/webformsession.js
@@ -152,7 +152,7 @@ WebFormSession.prototype.serverRequest = function (requestParams, callback, bloc
     if (this.blockingRequestInProgress) {
         return;
     }
-    this.blockingRequestInProgress = blocking;
+    this.blockingRequestInProgress = blocking || false;
     $.publish('session.block', blocking);
 
     this.numPendingRequests++;
@@ -227,8 +227,8 @@ WebFormSession.prototype.handleFailure = function(resp, action, textStatus, fail
         human_readable_message: errorMessage
     });
     this.onLoadingComplete();
-    $.publish('session.block', false);
-    this.blockingRequestInProgress = false;
+//    $.publish('session.block', false);
+//    this.blockingRequestInProgress = false;
 };
 
 /*

